.jshintrc

```diff
@@ -51,7 +51,7 @@
     "jquery"        : true,
 
     "nomen"         : false,
-    "onevar"        : true,
+    "onevar"        : false,
```

Comment:
Blah. I don't like this style and [it was removed in JSHint. Notice you can't find it](http://jshint.com/docs/options/).